index.js

```diff
@@ -59,6 +59,7 @@ UglifyJSOptimizer.prototype.optimize = function(args, callback) {
       data: optimized.code,
       map: optimized.map
     } : {data: optimized.code};
+    result.data = result.data.replace(/\n\/\/# sourceMappingURL=\S+$/, '');
     callback(null, result);
```

Comment:
Why don't you just put this on the next line as the arg sent to callback instead of mutating in place first?